HelloWorldFunction/src/main/java/helloworld/App.java

```diff
@@ -22,102 +22,109 @@
+                                                                                                // AmazonServiceException
     {
 
-        HashMap<String,AttributeValue> key_to_get =
-        new HashMap<String,AttributeValue>();
-    
+        HashMap<String, AttributeValue> key_to_get = new HashMap<String, AttributeValue>();
+
         key_to_get.put(PrimaryKey, new AttributeValue(Value));
-        
+
         GetItemRequest request = null;
 
-        request = new GetItemRequest()
-            .withKey(key_to_get)
-            .withTableName(TableName);
-        
-        final AmazonDynamoDB ddb = AmazonDynamoDBClientBuilder.defaultClient();
-        
+        request = new GetItemRequest().withKey(key_to_get).withTableName(TableName);
+
+        AmazonDynamoDB ddb = AmazonDynamoDBClientBuilder.defaultClient();
+
         try {
-            Map<String,AttributeValue> returned_item =
-            ddb.getItem(request).getItem();
+            Map<String, AttributeValue> returned_item = ddb.getItem(request).getItem();
             if (returned_item != null) {
                 Set<String> keys = returned_item.keySet();
-                String output;
-                for (String key : keys) {
-                    System.out.format("%s: %s\n",
-                            key, returned_item.get(key).toString());
+                //String output;
+                for ( String key : keys) {
+                    System.out.format("%s: %s\n", key, returned_item.get(key).toString());
                 }
                 return returned_item.toString();
-                
+
             } else {
                 System.out.format("No item found with the key %s!\n", Value);
             }
         } catch (AmazonServiceException e) {
             System.err.println(e.getErrorMessage());
-            //System.exit(1);
-            return(e.getErrorMessage());
+            // System.exit(1);
+            return (e.getErrorMessage());
         }
-        return "";
-        
+        return "successfully put item";
+
     }
 
-    private String PutItem(String TableName,String PrimaryKey, String PrimaryKeyValue, String Key, String Value) //throws AmazonServiceException,ResourceNotFoundException
+    private String PutItem(String TableName, String PrimaryKey, String PrimaryKeyValue, String Key, String Value) // throws AmazonServiceException,ResourceNotFoundException
     {
-        HashMap<String,AttributeValue> item_values =
-        new HashMap<String,AttributeValue>();
-    
+        HashMap<String, AttributeValue> item_values = new HashMap<String, AttributeValue>();
+
         item_values.put(PrimaryKey, new AttributeValue(PrimaryKeyValue));
         item_values.put(Key, new AttributeValue(Value));
-        
-        final AmazonDynamoDB ddb = AmazonDynamoDBClientBuilder.defaultClient();
-        
+
+        AmazonDynamoDB ddb = AmazonDynamoDBClientBuilder.defaultClient();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you don’t explicitly set an AWS region, the SDK consults the default region provider chain to try and determine the region to use.  However, the resulting delay might lead to a poor cold-start. If code portability is not an issue, consider explicitly setting the AWS region using the *withRegion()* method.

[Learn more](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#automatically-determine-the-aws-region-from-the-environment)